jp/event_summaries_api.md

```diff
@@ -16,18 +16,18 @@ events: [   # could maybe be a lone top-level array if there are no other fields
 Common `EVENT_DATA`:
 
 ```coffee
-  title: 'Chapter 2 Reading'
-  starts: timestamp
-  ends: timestamp
-  trouble: false
-  type: 'plan'   # distinguish plans from tasks from basic events
-  subtype: 'reading'  # useful to display reading plan icon or something
-  id: 42  # to be used with GET /api/plans/42
-```
+  start_at: timestamp
+  end_at: timestamp
+  is_trouble: false
```

Comment:
This `is_trouble` is outside of the payload, but below it is inside the payload.  Which did you intend to be the right way?